tests/core/ckeditor/manual/assetscachekeycdn.html

```diff
@@ -0,0 +1,114 @@
+<div id="editor"></div>
+<button disabled id="loadButton">Load external editor</button>
+<div id="loading_box" style="margin: 20px 0"></div>
+<div id="editor_cdn">
+	<p>A simple editor</p>
+</div>
+<div id="fake_observable_item"></div>
+
+<script>
+	( function () {
+		if ( bender.tools.env.mobile || ( CKEDITOR.env.ie && CKEDITOR.env.version < 11 ) ) {
+			return bender.ignore();
+		}
+
+		// 1. Load local editor by default with bender.
+		// 2. Remove existing instance and add script with editor from CDN.
+		// 3. Wait for the new editor to be loaded from CDN and initialize it.
+		// 4. Add icons in IE <11 from local.
+		var editorVersion = '',
+			path = getCKEditorCORSLink( window.location ),
+			loadButton = CKEDITOR.document.getById( 'loadButton' );
+
+		CKEDITOR.replace( 'editor', {
+			// Set fake element as observableParent due to removing existing instance of the editor.
+			observableParent: document.getElementById( 'fake_observable_item' )
+		} );
+
+		CKEDITOR.once( 'instanceReady', function() {
+			loadButton.$.disabled = false;
+			loadButton.once( 'click', loadExternalEditor );
+		} );
+
+		function loadExternalEditor() {
+			var scripts = document.querySelectorAll( 'script' ),
+				styles = document.querySelectorAll( 'style' ),
+				head = CKEDITOR.document.getHead();
+
+			CKEDITOR.tools.array.forEach( styles, function ( style ) {
+				style.parentNode.removeChild( style );
+			} );
+
+			CKEDITOR.tools.array.forEach( scripts, function ( script ) {
+				script.parentNode.removeChild( script );
+			} );
+
+			CKEDITOR.instances.editor.destroy();
+			window.CKEDITOR = null;
+
+			var newScript = document.createElement( 'script' ),
+				loadingBox = document.getElementById( 'loading_box' ),
+				loadingText = 'Simulating loading editor from CDN';
+
+			newScript.type = 'text/javascript';
+			newScript.src = path + '/ckeditor.js';
+			document.getElementsByTagName( 'head' )[ 0 ].appendChild( newScript );
+
+			var interval = setInterval( function () {
+				try {
+					if ( CKEDITOR.status === 'loaded' ) {
+						clearInterval( interval );
+
+						CKEDITOR.replace( 'editor_cdn', {
+							removePlugins: 'scayt,exportpdf'
+						} );
+						loadingBox.innerText = editorVersion + ' editor version was loaded successfully!';
+
+						// There is problem in IE <11 with receiving icons from githack so we must add it manually.
+						if ( CKEDITOR.env.ie && CKEDITOR.env.version < 11 ) {
+							CKEDITOR.once( 'instanceReady', function () {
+								fixButtonsIcons();
+							} );
+						}
```

Comment:
This test ignores IE < 11. Either we should remove the ignore condition or this code?